### ISSUE TITLE
Wrong count of csv files left to be loaded in log file 

### ISSUE THREAD

**Issue by erssebaggala** [bodastage/bts-ce-lite#61]
Operating System: Windows, Linux. Mac OSX
Boda-Lite Version: v0.2.2

During loading, the number of csv files that have not loaded is wrong. This happens when the csv file name has no corresponding table and the error below is added to the log

```
...
...[error] Pool_Connect_Query: error: relation "huawei_cm.BRD_MOD" does not exist
...[info] Skipping loading of BRD_MOD
...
```

**Comment by erssebaggala** [comment]
The issue caused by a bug at line below: https://github.com/bodastage/bts-ce-lite/blob/c593e2b982ba63d2c35a21b5a5e06aded352f114/background/background-utils.js#L441

csvFileCount should be decreased; not incremented!

**Comment by erssebaggala** [comment]
Fixed in PR #69
